settings_sample.py

```diff
@@ -3,3 +3,6 @@
 DB_NAME = ''
 USER = ''
 PASSWD = ''
+NODE_WHITELIST_ENABLED = False
+# IP addresses separated by comma
+NODE_WHITELISTED_IP_ADDRESSES = ''
```

Comment:
Linter is complaining about missing newline. You can enable PyCharm to finish all lines automatically in preferences (ensure line feed...):

<img width="1061" alt="screen shot 2018-02-15 at 19 43 05" src="https://user-images.githubusercontent.com/1409983/36271888-98df4036-1288-11e8-91b6-dca1b958c1fc.png">

Reply:
thanks